spotlight-client/src/OtherNarrativeLinksPreview/OtherNarrativeLinksPreview.tsx

```diff
@@ -141,58 +135,67 @@ const RacialDisparitiesPreview = observer(
     return (
       <ModelHydrator model={narrative}>
         <>
-          <ChartTitle>Population by Race/Ethnicity</ChartTitle>
           {narrative.populationDataSeries && (
-            <BarChartPair data={narrative.populationDataSeries} preview />
+            <ChartPreview>
+              <BarChartPair data={narrative.populationDataSeries} preview />
+            </ChartPreview>
           )}
+          <ChartTitle>Population by Race/Ethnicity</ChartTitle>
         </>
       </ModelHydrator>
     );
   }
 );
 
-const NarrativeLink: React.FC<{
-  narrative: Narrative;
-  tenantId: TenantId;
-}> = observer(({ narrative, tenantId }) => {
-  const [animationStyles, setAnimationStyles] = useSpring(() => ({
-    opacity: 0,
-    from: { opacity: 0 },
-  }));
+const NarrativeTabs: React.FC<{
+  narratives: Narrative[];
+  onTabChange: (selectedTab: NarrativeTypeId) => void;
+}> = observer(({ narratives, onTabChange }) => {
+  const [selectedTab, selectTab] = React.useState<NarrativeTypeId>(
+    DEFAULT_SELECTED_TAB
+  );
+  const [tabIndex, setTabIndex] = React.useState(
+    PREVIEW_ORDER.indexOf(DEFAULT_SELECTED_TAB)
+  );
+
+  React.useEffect(() => {
+    const nextIndex = (tabIndex + 1) % (PREVIEW_ORDER.length - 1);
+
+    const timer = setTimeout(() => {
+      setTabIndex(nextIndex);
+      selectTab(PREVIEW_ORDER[nextIndex]);
+    }, 5000);
```

Comment:
I personally think, that 5 seconds is not enough for user to try interact with preview. Maybe double it? Try it for yourself @hobuobi @terryttsai